tools/build_release.py

```diff
@@ -86,6 +121,7 @@ def main():
     if args.upload:
         logging.info('Uploading to PyPI')
         complete_twine_cmd = twine_cmd + ['upload']
+        drivers_to_upload = [driver for driver in drivers_to_update if driver != 'nifake']
```

Comment:
In order to support releasing individual drivers, we need to look at args.drivers.